client/src/app/site/pages/meetings/pages/participants/pages/participant-detail/components/participant-detail-view/participant-detail-view.component.ts

```diff
@@ -339,11 +339,12 @@ export class ParticipantDetailViewComponent extends BaseMeetingComponent {
             this.goToAllUsers();
         }
 
-        if (edit) {
-            await this.modelRequestService.waitSubscriptionReady(PARTICIPANT_DETAIL_SUBSCRIPTION);
-            setTimeout(() => (this._userFormLoaded = true), 1000);
-        } else {
+        if (!edit) {
             this._userFormLoaded = false;
+        } else {
+            this.modelRequestService
+                .waitSubscriptionReady(PARTICIPANT_DETAIL_SUBSCRIPTION)
+                .then(() => setTimeout(() => (this._userFormLoaded = true), 1000));
```

Comment:
Your previous version of this was better

Reply:
Used the older version now.